pipelines/mill_family.py

```
@@ -0,0 +1,592 @@
+Nothing here reads or writes a corpus tree: callers feed decoded records in and
+get findings back.  See ``census.py`` (reporting) and ``curate_agentic.py``
+(quarantine) for the two in-tree callers.
+"""
+
+from __future__ import annotations
+
+import re
+from collections import Counter, defaultdict
+from dataclasses import dataclass, field
+from fractions import Fraction
+from typing import Any, Hashable, Iterable, Mapping
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Hashable`, `Iterable`, `Mapping`

Import from `collections.abc`

(UP035)